src/semgrep_agent/main.py

```diff
@@ -24,6 +24,22 @@
 from semgrep_agent.utils import print_sh_error_info
 
 
+ALL_MANUAL_ENV_VARS = {
+    "SEMGREP_BRANCH": "The scanned branch ref",
+    "SEMGREP_COMMIT": "The scanned commit SHA",
+    "SEMGREP_JOB_URL": "The URL of the CI job that ran this scan, if any",
+    "SEMGREP_PR_ID": "The pull or merge request ID, if any",
+    "SEMGREP_PR_TITLE": "The pull or merge request title, if any",
+    "SEMGREP_REPO_NAME": "The name of this repository",
+    "SEMGREP_REPO_URL": "This repository's code URL",
```

Comment:
```suggestion
    "SEMGREP_REPO_URL": "The URL of this repository",
```

for consistency with JOB_URL

Reply:
I think that's confusing though, right? b/c a "repository" isn't well-defined for this purpose.